packages/components/pie-textarea/src/index.ts

```diff
@@ -20,16 +21,25 @@ export class PieTextarea extends RtlMixin(LitElement) implements TextareaProps {
     @property({ type: Boolean, reflect: true })
     public disabled?: TextareaProps['disabled'];
 
+    @property({ type: String })
```

Comment:
Please can you use the `validPropertyValues` decorator from `pie-webc-core`?

Here's an example of how it's used in `pie-button` https://github.com/justeattakeaway/pie/blob/main/packages/components/pie-button/src/index.ts#L55